.github/workflows/release-lfx.yml

```
@@ -0,0 +1,389 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: lfx-dist
+          path: dist/
+
+      - name: Generate release notes
+        id: notes
+        run: |
```

actionlint (1.7.7)

property "validate-version" is not defined in object type {build-docker: {outputs: {}; result: string}; release-lfx: {outputs: {version: string}; result: string}}

(expression)